src/ng/services/table.js

```diff
@@ -1,22 +1,143 @@
+class Dom {
+	column() {
+		return {cells: []};
+	}
+
+	row() {
+		return {cells: []};
+	}
+
+	cell() {
+		return null;
+	}
+
+	get view() {
+		return null;
+	}
+}
+class TableDom extends Dom {
+	constructor(element) {
+		super();
+		this.element = element;
+	}
+
+	static get empty() {
+		return new Dom();
+	}
+
+	get view() {
+		return this.element;
+	}
+
+	column(index) {
+		return new ColumnDom(this.element, index);
+	}
```

Comment:
Column and row should have the same behavior on checking index. For row if index not correct - null is return, for column ColumnDom.